framework/test/randoop_single_coverage.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+################################################################################
+#
+# This script generates coverage data for Randoop generated tests over a single defects4j test.
+#
+################################################################################
+# Import helper subroutines and variables, and init Defects4J
+source test.include
+init
+
+# change 0 to 1 for extra logging info and to save temp files
+debug=0
+
+# master coverage file
+master_coverage=$TMP_DIR/coverage
+
+# Directory for Randoop test suites
+randoop_dir=$TMP_DIR/randoop
+
+# Generate tests for Lang-4
+# Mockito #1 and #3 don't work; problem finding classes in byte-buddy?
+# Thus the strange hack below
```

Comment:
This seems related to issue #89, which we have fixed. Can we remove this special case?